pkg/alertmanager/operator.go

```diff
@@ -53,7 +54,8 @@ import (
 )
 
 const (
-	resyncPeriod = 5 * time.Minute
+	resyncPeriod                   = 5 * time.Minute
+	prometheusOperatorFieldManager = "PrometheusOperator"
```

Comment:
nit: It doesn't hurt, but if it has only one usage I wouldn't bother creating a var for it. Do you plan to use this in any follow-up PRs? Don't bother changing it back just because of this, just curious.

Reply:
The same change will be applied to the Prometheus and Thanos controllers so we'll need a constant in the end.